client/src/components/participant-details/track-graduation.js

```diff
@@ -162,6 +162,7 @@ export const TrackGraduation = (props) => {
                   await handleArchive(props?.participant?.id, values, openToast);
                   fetchData();
                 }}
+                participantId={props?.participant?.id}
```

Comment:
Can we create a variable for `props?.participant?.id`? Since, it's used in multiple places
```
await handleArchive(props?.participant?.id, values, openToast);
```